libnavigation-core/api/current.txt

```diff
@@ -1,6 +1,9 @@
 // Signature format: 3.0
 package com.mapbox.navigation.core {
 
+  public final class HistoryRecordingStateHandlerKt {
```

Comment:
Ping, think this is not supposed to be here

Reply:
No, it's OK. Discussed it here: https://github.com/mapbox/mapbox-navigation-android/issues/6051.